Cargo.toml

```diff
@@ -1,17 +1,13 @@
 [workspace]
-members = [
-    "block-producer",
-    "node",
-    "proto",
-    "rpc",
-    "store",
-    "utils",
-]
+members = ["block-producer", "node", "proto", "rpc", "store", "utils"]
 resolver = "2"
 
 [workspace.dependencies]
-miden-crypto = { package = "miden-crypto", git = "https://github.com/0xPolygonMiden/crypto.git", branch = "next", features = ["std"] }
-miden_objects = { package = "miden-objects", git = "https://github.com/0xPolygonMiden/miden-base.git", branch = "main", default-features = false }
+miden-crypto = { package = "miden-crypto", git = "https://github.com/0xPolygonMiden/crypto.git", branch = "next", features = [
+    "std",
+] }
+miden-lib = { package = "miden-lib", git = "https://github.com/0xPolygonMiden/miden-base.git", branch = "main" }
+miden_objects = { package = "miden-objects", git = "https://github.com/0xPolygonMiden/miden-base.git", branch = "main", features = ["serde"] }
```

Comment:
Are we still using `serde` feature somewhere?

Reply:
removed